src/reanimated2/UpdateProps.js

```diff
@@ -23,8 +23,9 @@ const colorProps = [
 
 const ColorProperties = makeShareable(colorProps);
 
-export default function updateProps(viewTag, updates, maybeViewRef) {
+export default function updateProps(viewDescriptor, updates, maybeViewRef) {
   'worklet';
+  const viewName = viewDescriptor?.value?.name || 'RCTView';
```

Comment:
under what circumstances view name cannot be set while the tag is? Maybe we can just use the same pattern here as with the tag?

Reply:
yes, they're assigned together and we expect view name 100% to be correct, fixed 9d309f2